ims_lti_py/request_validator.py

```diff
@@ -36,10 +37,13 @@ def is_valid_request(self, request, parameters={},
                 headers=headers,
                 parameters=parameters)
 
+            if isinstance(oauth_request['oauth_signature'], six.string_types):
+                oauth_request['oauth_signature'] = oauth_request['oauth_signature'].encode('utf-8')
```

Comment:
Everything in this PR was autogenerated except for this. This is needed to ensure the signature comparison happens correctly. Things worked previously because in python2:
```
>>> 'foo' == b'foo'
True
```